src/cmd/script.rs

```diff
@@ -47,6 +47,13 @@ pub fn new<'a, 'b>() -> App<'a, 'b> {
                 .long("relative-path")
                 .help("Use relative paths (flist generation only)"),
         )
+        .arg(
+            Arg::with_name("defines")
+                .long("defines")
+                .help("Pass an additional define to all source files")
+                .takes_value(true)
+                .multiple(true),
+        )
```

Comment:
Very nice idea! Also setting `.multiple(true)` is smart, because then you can pass things like `--defines A=0 B=1 C=2` if there is no ambiguity with other options. Small nitpick: How about calling the long option `--define` (singular), and adding `.short("D")` such that you can also write `-D A=0` (or even `-DA=0` thanks to clap).

Reply:
Yes, I think adding a `-D` option is a no-brainer.